### PR TITLE
[Release] Update index.json for extension [ connectedk8s ] [ mesh ] [ db-up ] [ k8s-extension ] [ aks-preview ] 

### PR DESCRIPTION
**Pull request by azclibot**
[Release] Update index.json for extension [ connectedk8s ] [ mesh ] [ db-up ] [ k8s-extension ] [ aks-preview ] 

 Triggered by Azure CLI Extensions Release Pipeline - ADO_BUILD_ID=904663

Last commit against master: https://github.com/Azure/azure-cli-extensions/commit/f19ddbead01323e7cc4c69ab2ab0d906aec8d9eb

### CODE REVIEW

**Comment by yonzhan** [comment]
connectedk8s

**Comment by alphaWizard** [comment]
@Juliehzl ,Why hasn't this(connectedk8s v1.1.4) been released yet? Any reason to hold this. Also, the code for connectedk8s v1.1.5 through [PR](https://github.com/Azure/azure-cli-extensions/pull/3407) has been merged.

**Comment by zhoxing-ms** [comment]
@alphaWizard Hi, because the release of this PR depends on the official release of azure-cli, and the official release time of azure-cli is 05-25, so please wait until tomorrow to merge this PR~

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**
